cvat/apps/annotation/labelme.py

```diff
@@ -29,7 +29,7 @@
 
 def dump_frame_anno(frame_annotation):
     from collections import defaultdict
-    from lxml import etree as ET
+    from lxml import etree as ET # nosec (we generate xml)
```

Comment:
The comment makes little sense for unfamiliar reader. Also, it'd be probably better to rephrase it with some accent on that fact of generation, like `xml is being generated here`.